solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -412,97 +412,100 @@ pub mod solana_ibc {
-            return Err(error::Error::InvalidTimeout.into());
-        }
-
-        let sequence = store
-            .get_next_sequence_send(&ibc::path::SeqSendPath::new(
-                &port_id,
-                &channel_id,
-            ))
-            .map_err(error::Error::ContextError)
-            .map_err(|err| error!((&err)))?;
-
-        let port_channel_pk = PortChannelPK::try_from(&port_id, &channel_id)
-            .map_err(|e| error::Error::ContextError(e.into()))?;
-
-        let channel_end = store
-            .borrow()
-            .private
-            .port_channel
-            .get(&port_channel_pk)
-            .ok_or(error::Error::Internal("Port channel not found"))?
-            .channel_end()
-            .map_err(|e| error::Error::ContextError(e.into()))?
-            .ok_or(error::Error::Internal("Channel end doesnt exist"))?;
-
-        channel_end
-            .verify_not_closed()
-            .map_err(|e| error::Error::ContextError(e.into()))?;
-
-        let conn_id_on_a = &channel_end.connection_hops()[0];
-
-        let conn_end_on_a = store
-            .connection_end(conn_id_on_a)
-            .map_err(error::Error::ContextError)?;
-
-        let client_id_on_a = conn_end_on_a.client_id();
-
-        let client_state_of_b_on_a = store
-            .client_state(client_id_on_a)
-            .map_err(error::Error::ContextError)?;
-
-        let status = client_state_of_b_on_a
-            .status(store.get_client_validation_context(), client_id_on_a)
-            .map_err(|e| error::Error::ContextError(e.into()))?;
-        if !status.is_active() {
-            return Err(error::Error::ContextError(
-                ibc::ClientError::ClientNotActive { status }.into(),
-            )
-            .into());
-        }
-
-        let packet = ibc::Packet {
-            seq_on_a: sequence,
-            port_id_on_a: port_id,
-            chan_id_on_a: channel_id,
-            port_id_on_b: channel_end.remote.port_id,
-            chan_id_on_b: channel_end.remote.channel_id.ok_or(
-                error::Error::Internal("Counterparty channel id doesnt exist"),
-            )?,
-            data,
-            timeout_height_on_b: timeout_height,
-            timeout_timestamp_on_b: timeout_timestamp,
-        };
-
-        if cfg!(test) || cfg!(feature = "mocks") {
-            ::ibc::core::channel::handler::send_packet_validate(
-                &store, &packet,
-            )
-            .map_err(error::Error::ContextError)
-            .map_err(|err| error!((&err)))?;
-        }
-
-        // Since we do all the checks present in validate above, there is no
-        // need to call validate again.  Hence validate is only called during
-        // tests.
-        ::ibc::core::channel::handler::send_packet_execute(&mut store, packet)
-            .map_err(error::Error::ContextError)
-            .map_err(|err| error!((&err)))
-    }
+    /// This method is removed since we dont support cross chain messaging yet
```

Comment:
Let’s just delete it. No need to keep it. We can look into history to get it back if we need to.

Reply:
ok lets do that then